c++/src/capnp/compat/json.h

```diff
@@ -313,6 +313,28 @@ class JsonCodec::Handler<T, Style::POINTER>: private JsonCodec::HandlerBase {
   friend class JsonCodec;
 };
 
+template <>
+class JsonCodec::Handler<DynamicValue>: private JsonCodec::HandlerBase {
```

Comment:
Done, see below. Would you like me to remove this commit, and its reversion, from the branch history?